ftp-proxy.ks.in

```diff
@@ -0,0 +1,22 @@
+#test name: basic-ftp
+url --url=@KSTEST_FTP_URL@
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
```

Comment:
I wonder if using some fragments could prevent duplication ? For example, I don't think the storage section is specific to this test & could be replaced by: https://github.com/rhinstaller/kickstart-tests/blob/master/fragments/shared/storage/default.ks

Or possibly the full common_no_payload ?
https://github.com/rhinstaller/kickstart-tests/blob/master/fragments/shared/common/common_no_payload.ks

Reply:
This is taken from the ftp test. Let's use it this way and I can take a look on the optimization of both in next PR.